### PR TITLE
Change 60FPS flag icon

### PR DESCRIPTION
**Pull request by JoanRiosiPla**
Updates the hue change to an icon change in 60fps

Resolves https://github.com/ScratchAddons/ScratchAddons/issues/6404

### Changes

Add flag URIs
Change updateFlag() to switch between icons instead of between hues.

### Reason for changes

30 FPS and 60 FPS are differentiated by colouring the green flag blue with a hueRotate, which may not be accessible for colour-blind users.

### Tests

It's tested and it works (Chrome).

### CODE REVIEW

File: addons/60fps/userscript.js [hunk 1]
```diff
@@ -7,14 +7,20 @@ export default async function ({ addon, console }) {
...
....20969 11.1262 1.8613 11.5729 0.75 12.4' fill='%234CBF56'/><path d='M0.75 2C1.8613 1.1729 3.20969 0.726196 4.595 0.726196C5.98031 0.726196 7.3287 1.1729 8.44 2C9.5513 2.8271 10.8997 3.27381 12.285 3.27381C13.6703 3.27381 15.0187 2.8271 16.13 2V12.4C15.0187 13.2271 13.6703 13.6738 12.285 13.6738C10.8997 13.6738 9.5513 13.2271 8.44 12.4C7.3287 11.5729 5.98031 11.1262 4.595 11.1262C3.20969 11.1262 1.8613 11.5729 0.75 12.4' stroke='%2345993D' stroke-linecap='round' stroke-linejoin='round'/><path d='M0.75 16.75V0.75' stroke='%2345993D' stroke-width='1.5' stroke-linecap='round' stroke-linejoin='round'/><path d='M9.60727 12.4029L4.04607 16.4893C3.71586 16.732 3.25 16.4962 3.25 16.0864L3.25 7.91358C3.25 7.50381 3.71586 7.26802 4.04607 7.51066L9.60727 11.5971C9.87918 11.7969 9.87918 12.2031 9.60727 12.4029Z' fill='%23D8DB4A' stroke='%23B5BF42'/><path d='M15.6073 12.4029L10.0461 16.4893C9.71586 16.732 9.25 16.4962 9.25 16.0864L9.25 7.91358C9.25 7.50381 9.71586 7.26802 10.0461 7.51066L15.6073 11.5971C15.8792 11.7969 15.8792 12.2031 15.6073 12.4029Z' fill='%23D8DB4A' stroke='%23B5BF42'/></g><defs><clipPath id='clip0_0_3'><rect width='16.63' height='17.5' fill='white'/></clipPath></defs></svg>"
```
Comment (Samq64) [inline addons/60fps/userscript.js:12]: Could the SVG be a separate file?

**Comment by Samq64** [comment]
Could the fast-forward part of the icon be more yellow? It's probably fine right now, but a little more contrast would be nice.

**Comment by DNin01** [comment]
Really nice! The fast-forward symbol in the icon could probably be a shade of yellowish orange so it's easier to see.

File: addons/60fps/userscript.js (hunk 1 above)
Comment (WorldLanguages) [reply addons/60fps/userscript.js:12]: `addon.self.dir + "/path.svg"`

File: addons/60fps/userscript.js [hunk 2]
```diff
@@ -7,14 +7,21 @@ export default async function ({ addon, console }) {
...
...20969 11.1262 1.8613 11.5729 0.75 12.4' fill='%234CBF56'/><path d='M0.75 2C1.8613 1.1729 3.20969 0.726196 4.595 0.726196C5.98031 0.726196 7.3287 1.1729 8.44 2C9.5513 2.8271 10.8997 3.27381 12.285 3.27381C13.6703 3.27381 15.0187 2.8271 16.13 2V12.4C15.0187 13.2271 13.6703 13.6738 12.285 13.6738C10.8997 13.6738 9.5513 13.2271 8.44 12.4C7.3287 11.5729 5.98031 11.1262 4.595 11.1262C3.20969 11.1262 1.8613 11.5729 0.75 12.4' stroke='%2345993D' stroke-linecap='round' stroke-linejoin='round'/><path d='M0.75 16.75V0.75' stroke='%2345993D' stroke-width='1.5' stroke-linecap='round' stroke-linejoin='round'/><path d='M9.60727 12.4029L4.04607 16.4893C3.71586 16.732 3.25 16.4962 3.25 16.0864L3.25 7.91358C3.25 7.50381 3.71586 7.26802 4.04607 7.51066L9.60727 11.5971C9.87918 11.7969 9.87918 12.2031 9.60727 12.4029Z' fill='%23D8DB4A' stroke='%23B5BF42'/><path d='M15.6073 12.4029L10.0461 16.4893C9.71586 16.732 9.25 16.4962 9.25 16.0864L9.25 7.91358C9.25 7.50381 9.71586 7.26802 10.0461 7.51066L15.6073 11.5971C15.8792 11.7969 15.8792 12.2031 15.6073 12.4029Z' fill='%23D8DB4A' stroke='%23B5BF42'/></g><defs><clipPath id='clip0_0_3'><rect width='16.63' height='17.5' fill='white'/></clipPath></defs></svg>";
```
Comment (DNin01) [inline addons/60fps/userscript.js:13]: The fast-forward symbol in the icon could probably be a shade of yellowish orange so it's easier to see.

Comment (JoanRiosiPla) [reply addons/60fps/userscript.js:13]: The flag could also be blue to not affect current users of the addon.

Comment (DNin01) [reply addons/60fps/userscript.js]: > The flag could also be blue to not affect current users of the addon.

And have the fast-forward symbol in addition to that, of course. That sounds good👍

**Review by DNin01 — APPROVED** [review]

**Review by Samq64 — APPROVED** [review]
I don't think the blue is necessary with the icon and the flags have a slightly different shape but neither is a big deal.

**Comment by BroJac5246** [comment]
I think we should keep the blue color, or make it purple. For users who aren't color-blind, this is still more convenient.

**Comment by WorldLanguages** [comment]
Icon is finished, will soon make some tiny changes to the code and the name of the files.

Gracias @JoanRiosiPla

**Review by WorldLanguages — APPROVED** [review]

**Pull request merged**